tests/providers/gcp/gcp_test.py

```diff
@@ -634,8 +692,67 @@ def testExecuteQuery(self, mock_gcl_api):
     self.assertEqual(FAKE_LOG_ENTRIES[0], query_logs[0])
 
 
+class GoogleCloudStorageTest(unittest.TestCase):
+  """Test Google Cloud Storage class."""
+  # pylint: disable=line-too-long
+
+  @typing.no_type_check
+  def testSplitGcsPath(self):
+    """Tests that GCS path split is correctly done."""
+    bucket, object_uri = gcp_storage.SplitGcsPath('gs://fake-bucket/fake-folder/fake-object')
+    self.assertEqual('fake-folder/fake-object', object_uri)
+    self.assertEqual('fake-bucket', bucket)
+
+  @typing.no_type_check
+  @mock.patch('libcloudforensics.providers.gcp.internal.storage.GoogleCloudStorage.GcsApi')
+  def testGetObjectMetadata(self, mock_gcs_api):
+    """Test GCS object Get operation."""
+    api_get_object = mock_gcs_api.return_value.objects.return_value.get
+    api_get_object.return_value.execute.return_value = MOCK_GCS_OBJECT_METADATA
+    get_results = FAKE_GCS.GetObjectMetadata('gs://Fake_Path')
+    self.assertEqual(MOCK_GCS_OBJECT_METADATA, get_results)
+    self.assertEqual('5555555555', get_results['size'])
+
+  @typing.no_type_check
+  @mock.patch('libcloudforensics.providers.gcp.internal.storage.GoogleCloudStorage.GcsApi')
+  def testGetMD5Object(self, mock_gcs_api):
+    """Test GCS object MD5 hash."""
+    api_get_object = mock_gcs_api.return_value.objects.return_value.get
+    api_get_object.return_value.execute.return_value = MOCK_GCS_OBJECT_METADATA
+    md5_base64 = FAKE_GCS.GetMD5Object('gs://Fake_Path')
+    self.assertEqual('fakehash', md5_base64)
+    md5_hex = FAKE_GCS.GetMD5Object('gs://Fake_Path', in_hex=True)
+    self.assertEqual(base64.b64decode('fakehash').hex(), md5_hex)
+
+
+class GoogleCloudBuildeTest(unittest.TestCase):
+  """Test Google Cloud Build class."""
+  # pylint: disable=line-too-long
+
+  @typing.no_type_check
+  @mock.patch('libcloudforensics.providers.gcp.internal.build.GoogleCloudBuild.GcbApi')
+  def testCreateBuild(self, mock_gcb_api):
+    """Test that Cloud Builds are correctly created."""
+    build_create_object = mock_gcb_api.return_value.projects.return_value.builds.return_value.create
+    build_create_object.return_value.execute.return_value = MOCK_GCB_BUILDS_CREATE
+    build_response = FAKE_GCB.CreateBuild({'Fake-Build_body': None})
+    self.assertEqual(build_response, MOCK_GCB_BUILDS_CREATE)
```

Comment:
`self.assertEqual(MOCK_GCB_BUILDS_CREATE, build_response)` to conform to `(expected, actual)`

Reply:
done